src/lib/components/composites/paper-components/PaperEntry.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+    import PaperInfo from "$lib/components/composites/paper-components/PaperInfo.svelte";
+    import { type Paper, ReviewDecision, ReviewProgress, type ReviewStatus } from "$lib/types";
+    import UserAvatar from "$lib/components/composites/user-avatar/UserAvatar.svelte";
+    import { goto } from "$app/navigation";
+
+    interface PaperEntryProps {
+        paper: Paper;
+        projectId: number;
+        showReviewStatus?: boolean;
+        onClick?: () => void;
+    }
+
+    const navigateToPaperView = () => goto(`/project/${projectId}/paper/${paper.id}`);
+
+    const {
+        paper,
+        projectId,
+        showReviewStatus = false,
+        onClick = navigateToPaperView,
+    }: PaperEntryProps = $props();
+
+    // Mapping of review status to border color of paper entry
+    const reviewStatusColor: Record<ReviewStatus, string> = {
+        [ReviewProgress.Unreviewed]: "border-unreviewed-gray",
+        [ReviewDecision.Accepted]: "border-accept-green",
+        [ReviewDecision.Maybe]: "border-maybe-yellow",
+        [ReviewDecision.Declined]: "border-decline-red",
+    };
+
+    const showStatus = () => showReviewStatus && paper.reviewData !== undefined;
+
+    let timeoutId: ReturnType<typeof setTimeout> | null = null;
+    /**
+     * Handles the click event of the paper entry component by checking
+     * whether it was a single click (so no further click after 250ms) or a double click
+     * and call the corresponding functions:
+     *  - single click => onClick() (possible overridden, otherwise navigateToPaperView())
+     *  - double click => navigateToPaperView() (default)
+     *
+     * @param e - the click event
+     */
+    const handleClick = (e: MouseEvent) => {
+        if (e.detail === 1) {
+            if (timeoutId !== null) clearTimeout(timeoutId);
+            timeoutId = setTimeout(() => onClick(), 250);
+        } else if (e.detail === 2) {
+            if (timeoutId !== null) clearTimeout(timeoutId);
+            navigateToPaperView();
+        }
+    };
+</script>
+
+<!--
+@component
+Container displaying important paper information, optionally with review information.
+
+This component shows the
+  - paper id
+  - title
+  - authors (or 'unknown authors' if none are specified)
+as well as, if not in review mode, the review information about this paper.
+
+Furthermore this component is clickable and navigates to the corresponding paper view,
+if the event handler is not overridden.
+
+Usage:
+```svelte
+    <PaperEntry paper={paper} projectId={1} showReviewStatus={true} />
+```
+-->
+<button
+    type="button"
+    class="flex flex-row items-center justify-end pe-4 gap-3 w-full border {showStatus()
+        ? 'border-l-0'
+        : ''} border-container-border-grey rounded-md hover:bg-container-border-grey/35"
```

Comment:
Optionally, you could use the `cn` utils method here to add additional style.
There's a similar case below.
```suggestion
    class={cn(
        "flex flex-row items-center justify-end pe-4 gap-3 w-full border border-container-border-grey rounded-md hover:bg-container-border-grey/35",
        isStatusVisible ? "border-l-0" : "",
    )}
```

Reply:
Theoretically I could and used It at the beginning, but personally I like the the current style more. 
But if you want to make it consistent, I can change it?

Reply:
No, that's up to you.